Sabersense Prop Documentation

Sabersense-CC opened the pull request:
I've added a sabersense folder to the prop page documentation with several pages of info on the main Sabersense Prop features.

Sabersense-CC committed:
Create _introduction.md

Sabersense-CC committed:
Update _introduction.md

I forgot to add a couple of features. Now done.

Sabersense-CC committed:
Create array_selector.md

Array Selection explained.

Sabersense-CC committed:
More detail included

Prof, this is all kind of a work in progress, but I need to GitHub editor to be able to check formatting. I'll message when everything is done if that's OK.

Sabersense-CC committed:
More tidying...

Sabersense-CC committed:
Create blade_id.md

Sabersense-CC committed:
Create restore_defaults.md

Sabersense-CC committed:
Create quote_force_track.md

Sabersense-CC committed:
More tidying...

Sabersense-CC committed:
More additions...

Sabersense-CC committed:
Tweak

Sabersense-CC committed:
Typos...

Sabersense-CC committed:
Punctuation...

Sabersense-CC committed:
Formatting

Sabersense-CC committed:
Just learned a new formatting trick...

Sabersense-CC committed:
Create button_clicker.md

profezzorn commented on props/sabersense/_introduction.md:
```diff
@@ -0,0 +1,32 @@
+## Button ‘Clicker’
+For hilts that have buttons with poor tactile feedback (KR’s Scavenger for instance, with the emitter wheel button setup), the system can be set to play custom click sounds to accompany button presses, aiding in system navigation.
+
+## System Reset Feature
```
Provide a link to the page that explains this feature.

Sabersense-CC replied:
Links added as appropriate throughout.

profezzorn reviewed (commented):
first pass...

Sabersense-CC committed:
First Pass fix.

Sabersense-CC committed:
First pass fix array selector

Sabersense-CC committed:
First pass fix Blade ID

Sabersense-CC committed:
First pass fix button_clicker

Sabersense-CC committed:
First pass fix quote_force_track

Sabersense-CC committed:
First pass fix restore_defaults

Sabersense-CC commented on props/sabersense/_introduction.md:
```diff
@@ -0,0 +1,32 @@
+# Introducing the Sabersense Button System
```
Added to each page.

Sabersense-CC committed:
Yikes - just saw code extracts indenting needed a tweak...

Now sorted.

Sabersense-CC committed:
Grammar and wording tweaks.

Sabersense-CC committed:
I thought of another implementation...

Switching effects player functionality on and off.

Sabersense-CC committed:
Tweaks and typo fixed...

Sabersense-CC committed:
Add instructions for Default Array and Blade Detect.

Sabersense-CC committed:
Update OS-7 compatibility Comments...

Sabersense-CC committed:
Update array_selector.md with new Blade Detect details

Updated to include details about using Blade Detect at the same time as SABERSENSE_ARRAY_SELECTOR. Other beta features that have been abandoned also removed from documentation.

Sabersense-CC committed:
Update blade_id.md - Fixed Typos.

Speeling mishtaks fizzed.

Sabersense-CC committed:
Update array_selector.md - BD link added

Sabersense-CC committed:
Update restore_defaults.md - Grammar Fix

Sabersense-CC committed:
Merge branch 'profezzorn:main' into patch-1

Sabersense-CC committed:
Create Controls.md

Work in progress - ignore for now....

Sabersense-CC committed:
Rename Controls.md to controls.md

Sabersense-CC committed:
Update and rename controls.md to 2_btn_controls.md

Sabersense-CC committed:
Create 1_btn_controls.md

Sabersense-CC committed:
Rename 1_btn_controls.md to controls_1_btn.md

Sabersense-CC committed:
Rename controls_1_btn.md to _controls_1_btn.md

Sabersense-CC committed:
Rename 2_btn_controls.md to _controls_2_btn.md

Sabersense-CC committed:
Rename _controls_1_btn.md to user_controls_1_btn.md

Sabersense-CC committed:
Rename _controls_2_btn.md to user_controls_2_btn.md

Sabersense-CC commented:
OK, I've been through everything, updated it and tidied it up, and added one and two button user controls. :)
All finished and good to go. :)

profezzorn commented on props/sabersense/_introduction.md:
```diff
@@ -0,0 +1,36 @@
+
+## Overview
+Built on Matthew McGeary’s SA22C setup, the Sabersense Prop File has been designed with simplicity in mind. It provides a saber control system that is intuitive, easy to use
+and consistent in terms of ‘rules’. It doesn’t require a ‘knack’ to access certain features, and tricky combinations like holding a button while simultaneously twisting or making other active gestures have been mostly removed. More complex features like Battle Mode are also not included.
```
no newlines in paragraphs

profezzorn commented on props/sabersense/array_selector.md:
```diff
@@ -0,0 +1,234 @@
+title: Array Selector
+---
+
+# Array Selector Explained
```
Why not just make this the title?
Looks weird to have two titles.

profezzorn commented on props/sabersense/array_selector.md:
```diff
@@ -0,0 +1,234 @@
+## Principles and Practice
+Array selection is a seemingly simple tool that is actually surprisingly powerful. The idea is that multiple different blade settings can be accessed at the press of a button, with simple config tweaks providing enormous options. Adjustments to the blade array can help build many different Array Selection setups, such as:
+
+- By specifying dummy blades, you can have a different number of effective blades on the same saber by switching arrays.
```
Hmm, I think * is normally used for bullet lists, not - ?

Sabersense-CC replied:
Fixed.

profezzorn reviewed (commented):
Didn't actually make it through all the files yet...

Sabersense-CC committed:
Update _introduction.md

All comments implemented.

Sabersense-CC commented on props/sabersense/array_selector.md:
```diff
@@ -0,0 +1,234 @@
+---
+title: Array Selector
```
Done.

Sabersense-CC committed:
Update array_selector.md

Suggested changes implemented.

Sabersense-CC committed:
Update blade_id.md

Doubled title removed.

Sabersense-CC committed:
Update button_clicker.md

Doubled title removed.

Sabersense-CC committed:
Update quote_force_track.md

Doubled title removed.

Sabersense-CC committed:
Update restore_defaults.md

Doubled title removed.

Sabersense-CC committed:
Update user_controls_1_btn.md

Doubled title removed.

Sabersense-CC committed:
Update user_controls_2_btn.md

Doubled title removed.

Sabersense-CC commented on props/sabersense/array_selector.md:
```diff
@@ -0,0 +1,234 @@
+# Array Selector Explained
+
+## Principles and Practice
+Array selection is a seemingly simple tool that is actually surprisingly powerful. The idea is that multiple different blade settings can be accessed at the press of a button, with simple config tweaks providing enormous options. Adjustments to the blade array can help build many different Array Selection setups, such as:
```
Reworded accordingly.

Sabersense-CC committed:
Default Array Feature added

Sabersense-CC committed:
Example tweak
